docs/examples/templating/returning_templates_jinja.py

```diff
@@ -6,9 +8,15 @@
 from litestar.template.config import TemplateConfig
 
 
-@get(path="/", sync_to_thread=False)
-def index(name: str) -> Template:
-    return Template(template_name="hello.html.jinja2", context={"name": name})
+@get(path="/{template_type: str}", sync_to_thread=False)
+def index(name: str, template_type: str | None) -> Template:
```

Comment:
Maybe the type for `template_type` can be a `Literal["file", "string"]`? This way the final returning with `template_str` set to `None` wouldn't be needed.

Reply:
I'm not sure I am understanding but I think this is not allowed per my failed tests and https://docs.litestar.dev/latest/usage/routing/parameters.html#supported-path-parameter-types

Reply:
Maybe it is too late for me 😅 If you mean in the func. signature then I can do this I think im just having a disconnect on if its no longer testing the Template being empty but will just throw an error on the `Literal` enums mismatching once it hits none? Again... may be sleepy 😪

Reply:
Aah my bad, I meant that you could try setting the type of `template_type` in the handler signature itself. Something like this:

```python
@get(path="/{template_type: str}", sync_to_thread=False)
def index(name: str, template_type: Literal["file", "string"]) -> Template:
    ...
```